src/mock/offer-mock.js

```diff
@@ -0,0 +1,22 @@
+import { getRandomArrayElement, createId, getRandomInt } from '../util.js';
+import { EVENT_TYPES } from './variablies.js';
+
+
+const OFFER_TITLES = ['Order meal', 'Infotainment system', 'Choose seats', 'Book a taxi at the arrival point', 'Wake up at a certain time'];
+
+const createOfferId = createId();
+
+const createOffer = () => ({
+  id: `${createOfferId()}`,
```

Comment:
тут тип более корректный
```suggestion
  id: crypto.randomUUID(),
```